salesagility/SuiteCRM
# Fix email layout in the Employees info.

Within the employees information view, the emails are joined on the same line, although they are seperate addresses.

## Description



This change allows the emails to be displayed on their own line.

## How To Test This

1) Navigate to the Employees module
2) Select one of the to view the details
3) View the Email addresses at the bottom
4) They should all be on seperate lines.

## Types of changes


### Final checklist
